package exception_handling;

public class Exception_Handling5 {

	
	public static void main(String[] args) {
		
		try {
			System.out.println("hello");
			System.out.println("hello");
			System.out.println("hello");
			System.out.println("hello");
			System.out.println("hello");
			System.out.println("hello");
			System.out.println("hello");
			System.out.println(5/0);
			System.out.println("hello");
			System.out.println("hello");
			System.out.println("hello");
			System.out.println("hello");
			System.out.println("hello");
			
		}catch(Exception e)
		
		{
			
			
			
			System.out.println(10/0);
		}
		finally
		{
			
			System.out.println("finally executed");
			
		}
		
		System.out.println("welcome India");
		
	}
}
